Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -89,20 +87,16 @@ public Long countDocuments(String query) throws DataException {
      *            condition as String
      * @return aggregate documents as JSONObject
      */
-    public JSONObject aggregateDocuments(String query, String aggregation) throws DataException {
+    public JsonObject aggregateDocuments(String query, String aggregation) throws DataException {
         SearchRestClient restClient = initiateRestClient();
-        JSONParser parser = new JSONParser();
 
         String response = restClient.aggregateDocuments(overrideNullQuery(query), aggregation);
         if (!response.equals("")) {
-            try {
-                JSONObject result = (JSONObject) parser.parse(response);
-                return (JSONObject) result.get("aggregations");
-            } catch (ParseException e) {
-                throw new DataException(e);
-            }
+            JsonReader jsonReader = Json.createReader(new StringReader(response));
```

Comment:
Same here. `JsonReader` is closeable so using a try-with-resources statement for closing the object is maybe a good idea.